README.md

```diff
@@ -116,24 +116,99 @@ For more information regarding the configuration of Open Liberty's logging capab
+        && mvn -f /opt/ol/wlp/usr/shared/resources/infinispan/pom.xml dependency:copy-dependencies -DoutputDirectory=/opt/ol/wlp/usr/shared/resources/infinispan \
+        && yum remove -y maven \
+        && rm -f /opt/ol/wlp/usr/shared/resources/infinispan/pom.xml \
+        && rm -f /opt/ol/wlp/usr/shared/resources/infinispan/jboss-transaction-api*.jar \
+        && rm -f /opt/ol/wlp/usr/shared/resources/infinispan/reactive-streams-*.jar \
+        && rm -f /opt/ol/wlp/usr/shared/resources/infinispan/rxjava-*.jar \
+        && rm -rf ~/.m2 \
+        && chown -R 1001:0 /opt/ol/wlp/usr/shared/resources/infinispan \
+        && chmod -R g+rw /opt/ol/wlp/usr/shared/resources/infinispan
+    USER 1001
+    
+    FROM openliberty/open-liberty:kernel-java8-openj9-ubi AS open-liberty-infinispan
+    
+    # Copy Infinispan client jars to Open Liberty shared resources
+    COPY --chown=1001:0 --from=infinispan-client /opt/ol/wlp/usr/shared/resources/infinispan /opt/ol/wlp/usr/shared/resources/infinispan
+    
+    # Instruct configure.sh to use Infinispan for session caching.
+    # This should be set to the Infinispan service name.
+    # TIP - Run the following oc/kubectl command with admin permissions to determine this value:
+    #       oc get infinispan -o jsonpath={.items[0].metadata.name}
+    ENV INFINISPAN_SERVICE_NAME=example-infinispan
+    
+    # Uncomment and set to override auto detected values.
+    # These are normally not needed if running in a Kubernetes environment.
+    # One such scenario would be the Infinispan deployment and Liberty deployment being in different namespaces/projects.
+    #ENV INFINISPAN_HOST=
+    #ENV INFINISPAN_PORT=
+    #ENV INFINISPAN_USER=
+    #ENV INFINISPAN_PASS=
+    
+    # This script will add the requested XML snippets and grow image to be fit-for-purpose
+    RUN configure.sh
+    ```
+
+    *  **Mount Infinispan Secret** - Finally, the Infinispan generated secret must be mounted as a volume under the mount point of `/platform/bindings/secret/` on Liberty containers. If the default location of `/platform/bindings/secret/` needs to be overridden, the `LIBERTY_INFINISPAN_SECRET_DIR` environment variable can be used. When using the Infinispan Operator, this secret is automatically generated as part of the Infinispan service with the name of `<INFINISPAN_CLUSTER_NAME>-generated-secret`. See the `volumes` and `volumeMounts` portions of the YAML below for an example of mounting this secret.
```

Comment:
in which namespace is the secret generated?   The app would have to be deployed in that same namespace.